artifacts/api-manager-artifacts/pattern-1/wso2am-p1-apim-1-conf.yaml

```diff
@@ -219,8 +219,8 @@ data:
     #port = 10005
 
     [[apim.throttling.url_group]]
-    traffic_manager_urls = ["tcp://localhost:9611"]
-    traffic_manager_auth_urls = ["ssl://localhost:9711"]
+    traffic_manager_urls = ["tcp://wso2-am-1-svc:9611"]
+    traffic_manager_auth_urls = ["ssl://wso2-am-1:9711"]
```

Comment:
This has to be wso2-am-1-svc:9711. I think we need to revisit service names that has used in here.